File: mavinokta-client/src/models/errorResponse.js
class Error {
  constructor(obj) {
    this.title = obj.title;
    this.detail = obj.detail;
  }

  title: string;
  detail: string;
}

export default class ErrorResponse {
  constructor(obj) {
    this.errorCode = obj.errorCode;
    this.errors = obj.errors;
    this.errorMessage = obj.errorMessage;
  }

  errorMessage: string;
  errorCode: number;
  errors: Array<Error>;
}

File: Otiport.Models/ErrorResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Otiport.Models
{
    public class ErrorResponse
    {
        public int ErrorCode { get; set; }
        public string ErrorMessage { get; set; }
        public List<Error> Errors { get; set; } = new List<Error>();
    }

    public class Error
    {
        public string Title { get; set; }
        public string Detail { get; set; }
    }
}
